src/common/src/config.rs

```diff
@@ -451,6 +451,10 @@ pub struct BatchConfig {
 
     #[serde(default, flatten)]
     pub unrecognized: Unrecognized<Self>,
+
+    #[serde(default = "default::batch::compute_runtime_worker_threads")]
+    /// compute runtime worker threads
+    pub compute_runtime_worker_threads: usize,
```

Comment:
```suggestion
    pub frontend_compute_runtime_worker_threads: usize,
```

I suggest to rename to this since we have also runtime for batch in compute node, adding this makes things more clear.

Reply:
Renamed.